server/controllers/nonproduction/db-persist.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const { Logger } = require('../../lib/logger');
+
+const ACL = {
+	user: false,
+	superuser: false,
+	superdooperuser: true
+};
+
+/* values for the post request
+* saved and deleted run in upsert history, started runs in mail contributor
+* you can also override any of the base dummy event items
+* expected body:
+{
+	"state": "[saved|deleted|started]",
+	"user": {
+		"first_name": stringValue,
+		"surname": stringValue,
+		"email": stringValue,
+		"id": valid User ID that is already in the syndication database (e.g. any dev with S1 product)
+	},
```

Comment:
[Suggestion] Add `role` to the expected user props. By default I believe we have 'user' level permissions, so we'll need to overwrite this locally when running the command. See the `ACL[user.role]` check on line 34.

Reply:
I've had a check and the res body values don't get written to locals. I solved this by updating my own role on the database to the `superdooperuser` setting, previously I'd just commented out this check and made sure not to commit it.

Reply:
Makes sense 👍